### ISSUE TITLE
Config not properly adding UHC worlds.

### ISSUE THREAD

**Issue by TheFakeIndian** [Ad5001/UHC#12]
Everytime I start a UHC it says, you are aren't in UHC world or UHC has already started.

**Comment by Ad5001** [comment]
Copy paste the example config into your UHC config and change the "uhc" by the name of your uhc world. Then restart your server.

**Comment by Wxngja** [comment]
Yes, it's still happening to me. I've been looking into the source code and couldn't find anything effecting this.
